ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -4,6 +4,28 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+trap 'FRC=$?; createUpgradeJunit' ERR EXIT TERM
+
+# Generate the Junit for upgrade
+function createUpgradeJunit() {
+    echo "Generating the Junit for upgrade"
+    if [[ "${FRC}" =  0  ]]; then
+      cat >"${ARTIFACT_DIR}/junit_upgrade.xml" <<EOF
```

Comment:
Add this line to the top of the XML:

`<?xml version='1.0' encoding='utf-8'?>`

Reply:
Thanks!! Updated.